src/main/java/com/google/api/codegen/config/GapicProductConfig.java

```diff
@@ -197,7 +197,14 @@ public static GapicProductConfig create(
     }
 
     // Toggle on/off proto annotations parsing.
-    ProtoParser protoParser = new ProtoParser(getDefaultLanguageFeatureConfig(language, null));
+    ProtoParser protoParser;
+    // TODO(andrealin): Expose command-line option for toggling proto annotations parsing.
+    if (configProto == null) {
```

Comment:
Nit: Maybe merge this with the configProto check on line 208? Don't feel that strongly though, please keep separate if you prefer.